Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -55,8 +55,9 @@ open class ChartDataSet: ChartBaseDataSet
     // MARK: - Data functions and accessors
 
     /// *
-    /// - note: Calls `notifyDataSetChanged()` after setting a new value.
-    /// - returns: The array of y-values that this DataSet represents.
+    ///
```

Comment:
do we need `///` here?

Reply:
The star can be removed.

Reply:
Ok, I just push a new commit to remove them.